src/thunderbots/software/ai/hl/stp/stp.cpp

```diff
@@ -0,0 +1,102 @@
+#include "ai/hl/stp/stp.h"
+
+#include <ai/hl/stp/play/play_factory.h>
+#include <munkres/munkres.h>
+
+#include <chrono>
+#include <exception>
+#include <random>
+
+#include "ai/hl/stp/play/play.h"
+#include "ai/hl/stp/tactic/tactic.h"
+#include "ai/intent/move_intent.h"
+#include "util/logger/init.h"
+
+STP::STP(long random_seed)
+    : random_number_generator(random_seed), uniform_distribution(1, 6)
```

Comment:
Is there any reason we would want to change the seed we use externally? I'm inclined to just have a static constant defined as part of the class for the seed.

Reply:
We want to be able to set the seed for testing, but otherwise want to get "true" randomness so the seed needs to be "random" too.

Reply:
Ok, I'm not super fussed either way, but why true randomness? Is there a disadvantage to being random but just the same way each time?